raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -960,6 +962,45 @@ public void testHandleBeginQuorumRequest(boolean withKip853Rpc) throws Exception
         );
     }
 
+    @Test
+    public void testHandleBeginQuorumRequestMoreEndpoints() throws Exception {
```

Comment:
is this a dupe of the test you added in KafkaRaftClientReconfigTest?

Reply:
ah I see, here we specify static voters

Reply:
Yeah. Similar test but we a different starting configuration.